AirCasting/Graph/Graph.swift

```diff
@@ -214,6 +225,48 @@ struct Graph: UIViewRepresentable {
             return chartDataEntry
         }) ?? []
         uiView.updateWithEntries(entries: entries, isAutozoomEnabled: isAutozoomEnabled)
+        
+        let limitLines = getLimitLines()
+        if limitLines != midnightLines {
+            midnightLines = limitLines
+            uiView.updateMidnightLines(with: limitLines)
```

Comment:
I'm wondering why the property `var midnightLines` is here in `Graph` instead of `AirCastingGraph`. I think it would be nicer to do so in `AirCastingGraph`, utilizing a `didSet` call. Note the `update` function can be made `private` in this scenario.

Inside `AirCastingGraph`:
``` Swift
var limitLines: [ChartLimitLine] = [] {
   didSet {
      guard oldValue != limitLines { return }
      updateLimitLines(with: limitLines)
   }
}

private updateLimitLines(with limitLines: [ChartLimitLine]) {
   ...
}
```

and here simply do:

```Swift
uiView.limitLines = getLimitLines()
```

As a side note: I also think anything named `midnight` should not be present in this view and be changed to `limit`. The reason is that it doesn't really need to be midnight, it can be any limit. Currently this midnights calculation logic is a stray - it has no clear and obvious home (it's not **really** UI, is it?). This is why layering would be beneficial. We'd have this nicely separated and unit tested inside some `Controller` object which would take care of fetching stuff from DB/Service and translating it into limits (midnights).